### PR TITLE
End round stats

### LINKED ISSUE #473: End round stats
Some stats are not being calculated I guess:
- [x] Produced power (hijack the SM power output proc somehow and start tracking power?)
- [x] Monkeys slain (add +1 to the stat in the monkey death code)
- [x] beepsky_beatings
- [x] ai_follow
- [x] trash_vented
- [x] cargo_profit

### PR DESCRIPTION
**Pull request by lmas**
- Fixes #473.
- Adds new SQL schema for storing end round stats.
- Stores said stats on end round.

This is part work of the apollo stats web page I'm currently working on. Yes I may have created a monster SQL query and yes, it was a bitch to debug.
I also fucked up the branch somehow so I couldn't save it on my own repo, @Kwask, after I had begun working on it.

Merry Xmas guys.

### CODE REVIEW

**Comment by lmas** [comment]
Hold on, got some more TODOs after I made the web page:
- [x] Clean up some var names in the sql.
- [x] ~~Maybe remove the break time var~~ Nope looks good.
- [x] Add stats for kicked/banned/arrested guys.
- [x] Make new table for ai laws and tie them with the round id.
- [x] New table for antags and tie in with a round.

**Comment by Kwask** [comment]
This doesn't modify any existing tables, correct? Those are always a pain to update :V

**Comment by lmas** [comment]
Nope, just a new table(s).

**Comment by lmas** [comment]
I lied. I updated the `death` table to add round_id to it. But I renamed it to `deaths` instead, so upgrading it will be just a simple `DROP TABLE death` and then just create the new table with the new name. 
A bonus is that deaths will now be reported on round end, only, and I can show more interesting stats on the web pages.

File: code/defines/procs/statistics.dm [hunk 1]
```diff
@@ -39,74 +39,6 @@ proc/sql_report_round_end()
...
-	establish_db_connection()
-	if(!dbcon.IsConnected())
-		log_game("SQL ERROR during death reporting. Failed to connect.")
-	else
-		var/DBQuery/query = dbcon.NewQuery("INSERT INTO death (name, byondkey, job, special, pod, tod, laname, lakey, gender, bruteloss, fireloss, brainloss, oxyloss, coord) VALUES ('[sqlname]', '[sqlkey]', '[sqljob]', '[sqlspecial]', '[sqlpod]', '[sqltime]', '[laname]', '[lakey]', '[H.gender]', [H.getBruteLoss()], [H.getFireLoss()], [H.brainloss], [H.getOxyLoss()], '[coord]')")
-		if(!query.Execute())
-			var/err = query.ErrorMsg()
-			log_game("SQL ERROR during death reporting. Error : \[[err]\]\n")
-
-
```
Comment (lmas) [inline code/defines/procs/statistics.dm]: Don't Repeat Yourself dammit! Keep things DRY!

**Comment by lmas** [comment]
Merge whenever you're ready to update the database.

**Comment by Kwask** [comment]
I'll plan on merging this tonight, when pop dies down a bit.

**Pull request merged**